TelePharo-Core.package/TlpRemotePharo.class/instance/applyChangesToServer.st

```diff
@@ -0,0 +1,6 @@
+requests
+applyChangesToServer
```

Comment:
I would name it applyChangesFromClient because we ask remote pharo to do it which is already a server. So we can have:
```Smalltalk
remotePharo applyChangesFromClient.
remotePharo applyChangesToClient.
```

Reply:
Done. But I used Github editor, I saw now was not a good idea because didn't change the name of the file... I will change using my Pharo Image.